reference-data-proxy/src/cronJobs/eStoreTermStoreAndBuyerFolder.cron.ts

```diff
@@ -1,3 +1,5 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
is there a way to fix this without disable?

Reply:
nope - because import doesn't work for this version of mongo :(